srcpkgs/volta/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'volta'
+pkgname=volta
+version=0.7.1
+revision=1
+archs="x86_64 i686"
```

Comment:
No musl support?

Reply:
Not sure if musl support is feasible, because the binaries that are added by volta might not work on musl systems. I need to test this before adding explicit musl support.

Reply:
Yeah, where do these binaries come from? What are they? Why can't they be built from source?

Reply:
Volta manages different node versions (among other things, but those are not binaries) In order to do that, it downloads them from an official repository.

Since it is intended to be run when you change into a directory, compiling every required node version from source would break the workflow.

Reply:
I'll set up a musl VM so I can test this.

Reply:
As I commented below, there is currently no musl support. As soon as volta supports it, I will add it to the template.